spec/services/idv/aamva_state_maintenance_window_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Idv::AamvaStateMaintenanceWindow do
+  let(:tz) { 'America/New_York' }
+
+  before do
+    travel_to Date.new(2024, 6, 2)
+  end
+
+  describe '#in_maintenance_window?' do
+    let(:state) { 'DC' }
+
+    subject { described_class.in_maintenance_window?(state) }
+
+    context 'for a state with a defined outage window' do
+      it 'is true during the maintenance window' do
+        Time.use_zone(tz) do
+          travel_to(Time.parse('June 2, 2024 at 1am')) do
+            expect(subject).to eq(true)
+          end
+        end
+      end
+
+      it 'is false outside of the maintenance window' do
+        Time.use_zone(tz) do
+          travel_to(Time.zone.parse('today 08:00')) do
+            expect(subject).to eq(false)
```

Comment:
I had this fail [here](https://gitlab.login.gov/lg/identity-idp/-/jobs/1388413). I think there is still some timezone oddness going on somewhere.

Even with `ZONEBIE_TZ="Brisbane" bers spec/services/idv/aamva_state_maintenance_window_spec.rb --seed 11439`, I could not reproduce the failure. It did seem to honor the Brisbane timezone, but tests passed.

I'm worried about introducing a flaky test, so I wouldn't mind an extra set of eyes.

Reply:
I am suspicious of the `today` reference. Brisbane is UTC +10 while I'm UTC -4, so it seems like there's room for "today" to be ambiguous. I can't actually reproduce that behavior, and it's running inside a block where I've explicitly set the timezone, but 🤷‍♂️ I will push up a change explicitly setting the time and date like I do on line 18.

Reply:
`Time.zone` will use the current timezone, which is what should get overridden, I would say that if you want to pin a timezone

I think you can make things clearear without the `Time.use_zone()` block by doing:

```ruby
utc = ActiveSupport::TimeZone['UTC']
utc.parse('....')
```

Reply:
or for this spec

```ruby
eastern_tz = ActiveSupport::TimeZone['America/New_York']
eastern_tz.parse('....')
```

Reply:
Oh! I wonder if rubocop did me dirty here. I had written `Time.parse` and it insisted I wanted `Time.zone.parse`. Tests still passed locally with it.

I like the pattern of explicitly doing `eastern_tz.parse` here.